core/src/main/java/org/apache/accumulo/core/data/RowRange.java

```diff
@@ -45,8 +45,91 @@ public RowRange() {
    * @param endRow ending row; set to null for positive infinity
    * @throws IllegalArgumentException if end row is before start row
    */
-  public RowRange(Text startRow, Text endRow) {
-    this(startRow, true, endRow, true);
+  public static RowRange open(Text startRow, Text endRow) {
```

Comment:
It kinda sucks that we're creating a new API with Text, as it would be nice to try to phase out that dependency on Hadoop types in our APIs. But, we don't have an alternative right now. You may want to consider offering overloaded methods for CharSequence, though, so users aren't forced to use Text if their endpoints are human-readable as Strings.

Reply:
Charsequence versions of the static methods would be nice.

Reply:
Addressed in a527552